test/parser/disk.spec.ts

```diff
@@ -2,7 +2,7 @@ import { describe, test } from 'vitest';
 import { t } from '../helpers/tap';
 
 import { Source } from '../../ast/source';
-import { StringLiteral, PathLiteral } from '../../ast/expr';
+import { StringLiteral, ShellLiteral } from '../../ast/expr';
```

Comment:
I may want to combine these tests with the load tests for general purpose "builtin" tests.